File: examples/different/submissions/accepted/different.cs
using System;

public class Program {
    public static void Main() {
        string line;
        while ((line = Console.ReadLine()) != null) {
            string[] split = line.Split(new char[] { ' ' }, StringSplitOptions.None);
            long a = Int64.Parse(split[0]), b = Int64.Parse(split[1]);
            Console.WriteLine(Math.Abs(a - b));
        }
    }
}

File: examples/different/submissions/accepted/different.ml
(* Get the i^th entry in the list, and convert it to an Int64. *)
let nth l i = (Int64.of_string (List.nth l i));;

try
    while true;
    do
        let line = read_line () in
        let tokens = Str.split (Str.regexp " ") line in
        let a = (nth tokens 0) in
        let b = (nth tokens 1) in
        let diff = Int64.abs (Int64.sub a b) in
            begin
                Printf.printf "%Ld\n" diff
            end;
    done;
with End_of_file -> ();;
